lm_eval/tasks/robustness/agi_eval/option_format_robustness_agieval.yaml

```diff
@@ -0,0 +1,37 @@
+group: option_format_robustness_agieval
+task:
+  - option_format_robustness_agieval_aqua_rat
+  - option_format_robustness_agieval_logiqa_en
+  - option_format_robustness_agieval_lsat_ar
+  - option_format_robustness_agieval_lsat_lr
+  - option_format_robustness_agieval_lsat_rc
+  - option_format_robustness_agieval_sat_en
+  - option_format_robustness_agieval_sat_math
+
+aggregate_metric_list:
+  - metric: prompt_0_macro_accuracy
```

Comment:
header +
what are these prompt_* here?
Does this one run eval on the default prompt?

Reply:
are these format options? if so, - why are they called prompt*?

Reply:
In terms of implementation format options and prompt robustness are the same task, with only the list of prompts being different, so I will remove the word "prompt" having only {index}_macro_accuracy, for the sake of avoiding confusion